research/picai/losses.py

```diff
@@ -0,0 +1,52 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class FocalLoss(nn.Module):
```

Comment:
I think I could also inherit from _Loss, but usually classes or functions with _ prepended are meant to be private and not accessed externally. To be consistent, I will change it to _Loss for this PR but I may update this in an upcoming one!

Reply:
That makes a lot of sense! 
Another suggestion for a future PR is to dedicate a loss module in fl4health where we keep all these loss implementations (there is another one for preprocessing.autoencoders for example). Do you think it would make sense to move FocalLoss under the main library as well?